Source/WebKit/UIProcess/WebProcessPool.cpp

```diff
@@ -1190,11 +1190,11 @@ void WebProcessPool::pageEndUsingWebsiteDataStore(WebPageProxyIdentifier pageID,
     }
 }
 
-bool WebProcessPool::hasPagesUsingWebsiteDataStore(WebsiteDataStore& dataStore) const
+HashSet<WebPageProxyIdentifier> WebProcessPool::pagesUsingWebsiteDataStore(WebsiteDataStore& dataStore) const
```

Comment:
May be nice to return a `const HashSet<WebPageProxyIdentifier>*` to avoid copying.

Reply:
That's if we *really* need the page IDs.